ui/app/routes/jobs/index.js

```diff
@@ -101,6 +193,8 @@ export default class IndexRoute extends Route.extend(
       Ember.testing ? 0 : DEFAULT_THROTTLE
     );
 
+    controller.parseFilter();
```

Comment:
Calls the "Change the ?filter=THING and THING and THING into dropdowns with pre-checked boxes" script on initial page load